src/components/account/account.tsx

```diff
@@ -57,7 +57,7 @@ export default function Account() {
                 </Avatar>
                 <Box sx={{ mr: 1, alignItems: 'center', verticalAlign: 'middle' }}>
                     <AccountAddress signer={appContext?.data.signer} address={address}/>
-                    <Box component="span" sx={{ display: { 'xs': 'none', 'md': 'inline-flex'}}}>
+                    <Box component="span" sx={{ cursor: 'ew-resize', display: { 'xs': 'none', 'md': 'inline-flex'}}}>
```

Comment:
I am going to revert this change as the _change_ cursor should only be shown when a conversion rate is available (that feature is configurable). Also we will redo this part of the component soon and then it will change anyways.